alloc.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2020, Michael Grunder <michael dot grunder at gmail dot com>
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *   * Redistributions of source code must retain the above copyright notice,
+ *     this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in the
+ *     documentation and/or other materials provided with the distribution.
+ *   * Neither the name of Redis nor the names of its contributors may be used
+ *     to endorse or promote products derived from this software without
+ *     specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "fmacros.h"
+#include "alloc.h"
```

Comment:
Isn't that deprecated?

Reply:
```
In file included from src/redisq_handler.h:5:0,
                 from src/am.cpp:14:
/usr/local/include/hiredis/hiredis.h:45:10: fatal error: alloc.h: No such file or directory
 #include "alloc.h" /* for allocation wrappers */
          ^~~~~~~~~
compilation terminated.
In file included from src/redisq_handler.h:5:0,
                 from src/redisq_handler.cpp:7:
/usr/local/include/hiredis/hiredis.h:45:10: fatal error: alloc.h: No such file or directory
 #include "alloc.h" /* for allocation wrappers */
          ^~~~~~~~~
compilation terminated.
Makefile:58: recipe for target 'am' failed
make: *** [am] Error 1
```

Reply:
Ahh, are you worried about a name conflict with the old "alloc.h" file?

Reply:
Yup. I though that was the problem at first. Anyway, you guys fixed the problem very fast. Thanks.